lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -143,14 +143,12 @@ export function setHeader(res: BaseResponse, name: string, value: string) {
 
 /**
  *
+ * @param config
```

Comment:
please also consider adding a test about this new setting.

Reply:
Hi, this is not new setting, sorry. It just my IDE change this
![image](https://user-images.githubusercontent.com/58660439/233563274-7c7fd99c-32bf-4cde-9def-2ddd71f12e53.png)
bc of things like that
![image](https://user-images.githubusercontent.com/58660439/233563413-1ccee6e0-a226-40d9-8ae4-3e380aaf9ea6.png)
automatically.

Reply:
Ah no. I meant that you should add a test about testing the new accessTokenPath config

Reply:
and where should I put this?

Reply:
Here are a few examples: https://github.com/supertokens/supertokens-node/blob/master/test/config.test.js

Reply:
Thanks, pushed tests